internal/integration/api/reset.go

```diff
@@ -114,17 +114,9 @@ func (suite *ResetSuite) TestResetNodeByNode() {
 }
 
 func (suite *ResetSuite) testResetNoGraceful(nodeType machine.Type) {
-	if !suite.Capabilities().SupportsReboot {
-		suite.T().Skip("cluster doesn't support reboot (and reset)")
-	}
-
-	if suite.Capabilities().TrustedBoot {
-		// this is because with trusted boot the talos.config is only applied and cannot be passed as kernel args
-		suite.T().Skip("cluster has trusted boot enabled, doesn't support reset")
-	}
-
-	if suite.Cluster == nil {
-		suite.T().Skip("without full cluster state reset test is not reliable (can't wait for cluster readiness in between resets)")
+	if !suite.Capabilities().SecureBooted {
```

Comment:
the condition is inverted here and other places

Reply:
ahh :face_with_head_bandage: , I'm dumb